package_control/library.py

```diff
@@ -101,14 +84,28 @@ def install(dest_root, src_dir, name, version, description, url, plat_specific):
             with open(os.path.join(dest_root, rel_dest), 'wb') as df:
                 df.write(sf.read())
 
-    di_record_path = os.path.join(dist_info_path, 'RECORD')
-    # Create an empty file so it shows up in its own file list
-    open(di_record_path, 'wb').close()
-    rf_contents = wheel.generate_record(
-        dest_root,
-        dist_info_dirname,
-        package_dir_names,
-        package_file_names
-    )
-    with open(di_record_path, 'w', encoding='utf-8') as fobj:
-        fobj.write(rf_contents)
+    dist_info.write_record(package_dir_names, package_file_names)
+
+
+def remove(install_root, name):
+    dist_info = find_dist_info_dir(install_root, name)
+
+    dir_names = set()
+
+    try:
+        record = dist_info.read_record()
+    except FileNotFoundError:
+        raise FileNotFoundError('Library {} not installed!'.format(name))
+
+    for file_name, file_hash, file_size in record:
+        abs_path = os.path.normpath(os.path.join(install_root, file_name))
+        dir_names.add(os.path.dirname(abs_path))
+        print("removing", abs_path)
+        # os.remove(abs_path)
```

Comment:
You probably meant to restore this. A debug log event might be useful regardless.